doc/nrf/releases/release-notes-latest.rst

```diff
@@ -141,6 +141,54 @@ Matter (Project CHIP)
 
   * Renamed occurrences of Project CHIP to Matter.
 
+Bluetooth mesh
+--------------
+
+* Added:
+
+  * Support for vendor-specific mesh model :ref:`bt_mesh_silvair_enocean_srv_readme`.
+  * A new API function ``bt_mesh_rpl_pending_store`` to manually store pending RPL entries in the persistent storage without waiting for the timeout.
+  * A :c:member:`bt_mesh_scene_entry::recall_complete` callback that is called for each model that has a scene entry when recalling a scene data is done.
+
+* Updated:
+
+  * Updated the :ref:`bt_mesh_light_xyl_srv_readme` to publish its state values that was loaded from flash after powering up.
```

Comment:
The word "was" is confusing or incorrect here. What was loaded from flash after powering up - the `bt_mesh_ligh_xyl_srv_readme` or the state values?

Here are two alternatives:

1) * Updated the :ref:`bt_mesh_light_xyl_srv_readme`  that was loaded from flash after powering up to publish its state values.
2) * Updated the :ref:`bt_mesh_light_xyl_srv_readme` to publish its state values that _were_ loaded from flash after powering up.